src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/InteropServices/CSharpDynamicInterfaceCastableImplementation.Fixer.cs

```diff
@@ -73,7 +71,7 @@ public sealed class CSharpDynamicInterfaceCastableImplementationFixer : DynamicI
                     return null;
                 }
                 var methodDecl = generator.MethodDeclaration(method);
```

Comment:
Could we rename this as `methodDeclaration`? I am not fan of abbreviations and it's not used consistently here (I know it was there before but there is enough changes to ask for it :)).

Reply:
Sure. Will do